tests/e2e/vsphere.go

```diff
@@ -763,15 +782,21 @@ func (vs *vSphere) getVsanClusterResource(ctx context.Context, forceRefresh ...b
 		}
 	} else {
 		framework.Logf("Found the default Cluster already set")
+		bootstrap()
```

Comment:
is will be of no use IMO, `cluster` var that is already declared will have the old `client` var

Reply:
Removed bootstrap()